### ISSUE TITLE
Gaussian processes

### ISSUE THREAD

**Issue by tingyuansen** [annayqho/TheCannon#45]
Is there any plan using Gaussian processes to do the continuum determination?

**Comment by davidwhogg** [comment]
The fit to the continuum using sines and cosines *is* a Gaussian process, actually!

**Comment by tingyuansen** [comment]
I agree that continuum using sines and cosines is equivalent to setting the 2-points correlation in Gaussian process to be some periodic function (e.g. sin(a(x-y))). Surely most, if not all regressions are some sort of Gaussian process, and the Cannon is a Gaussian process with 2-points correlation = a*x*y. I am just thinking a periodic Gaussian process is surely less robust than something like square exponential in spectrum continuum.
